examples/ChatQnA/deploy/k8s_helm.md

```diff
@@ -0,0 +1,497 @@
+	```bash
+    kubectl describe pod <pod-name>
+    ```
+For example, if the status of the TGI service does not show 'Running', describe the pod using the name from the above table:
+```bash
+kubectl describe pod chatqna-tgi-778bb6598f-cv5cg
+```
+or check logs using:
+```bash
+kubectl logs chatqna-tgi-778bb6598f-cv5cg
+```
+
+## Interacting with ChatQnA deployment
+This section will walk you through what are the different ways to interact with
+the microservices deployed
+
+Before starting the validation of microservices, check the network configuration of services using:
+```bash
+    kubectl get svc
+```
+   This command will display a list of services along with their network-related details such as cluster IP and ports. 
+ ```
+NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+chatqna                   ClusterIP   10.108.186.198   <none>        8888/TCP            8m16s
+chatqna-chatqna-ui        ClusterIP   10.102.80.123    <none>        5173/TCP            8m16s
+chatqna-data-prep         ClusterIP   10.110.143.212   <none>        6007/TCP            8m16s
+chatqna-nginx             NodePort    10.100.224.12    <none>        80:30304/TCP        8m16s
+chatqna-redis-vector-db   ClusterIP   10.205.9.19      <none>        6379/TCP,8001/TCP   8m16s
+chatqna-retriever-usvc    ClusterIP   10.202.3.15      <none>        7000/TCP            8m16s
+chatqna-tei               ClusterIP   10.105.204.12    <none>        80/TCP              8m16s
+chatqna-teirerank         ClusterIP   10.115.146.21    <none>        80/TCP              8m16s
+chatqna-tgi               ClusterIP   10.108.195.244   <none>        80/TCP              8m16s
+kubernetes                ClusterIP   10.92.0.100      <none>        443/TCP             11d
+ ```
+   To begin port forwarding, which maps a service's port from the cluster to local host for testing, use:
+ ```bash
+    kubectl port-forward svc/[service-name] [local-port]:[service-port]
+   ```
+   Replace `[service-name]`, `[local-port]`, and `[service-port]` with the appropriate values from your services list (as shown in the output given by `kubectl get svc`). This setup enables interaction with the microservice directly from the local machine. In another terminal, use `curl` commands to test the functionality and response of the service.
+
+Use `ctrl+c` to end the port-forwarding to test other services.
+
+
+### MegaService Before RAG Dataprep
+
+Use the following command to forward traffic from your local machine to the service running in the Kubernetes cluster:
+```bash
+kubectl port-forward svc/chatqna 8888:8888
+```
+Follow the below steps in a different terminal.
+
+```
+curl http://localhost:8888/v1/chatqna -H "Content-Type: application/json" -d '{
+     "model": "Intel/neural-chat-7b-v3-3",
+     "messages": "What is the revenue of Nike in 2023?"
+     }'
```

Comment:
Query specified here seems outdated based on answer?

Reply:
Getting the following error when running this query:

```
ubuntu@ip-172-31-7-63:~/.minikube$ curl http://localhost:8888/v1/chatqna -H "Content-Type: application/json" -d '{
     "model": "Intel/neural-chat-7b-v3-3",
     "messages": "What is the revenue of Nike in 2023?"
     }'
curl: (7) Failed to connect to localhost port 8888 after 0 ms: Couldn't connect to server
```

Reply:
@eero-t Fixed the query